.github/workflows/set-automatic-deploys-enabled.yaml

```
@@ -86,7 +86,7 @@ jobs:
           RESET_IMAGE_TAG: ${{ inputs.resetImageTag }}
         run: |
           IMAGE_TAG=""
-          if [ "${RESET_IMAGE_TAG}" ]; then
+          if [ "${RESET_IMAGE_TAG}" = "true" ]; then
```

nit: for consistency with the rest of the file, might as well use `[[` (i.e. the bash builtin, since we're already using bash-only stuff elsewhere in this script)

```suggestion
          if [ "${RESET_IMAGE_TAG}" = "true" ]; then
```

---

cool, thanks